packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx

```diff
@@ -39,6 +39,8 @@ interface TreeNodeRendererOwnProps {
   actionButtonsClassName?: string;
   /** Tree node size. Should match the size passed to `TreeRenderer` component. */
   size?: "default" | "small";
+  /** Configures filter buttons visibility. Defaults to `show-on-hover`*/
```

Comment:
I think we could do better with the docs - explain in more detail what each option means. Things worth mentioning:
- setting to `hide` will still show the buttons if the node is filtered,
- setting to `show-on-hover` will also show on focus.

Similarly, I'd like a more detailed explanation in the changeset file.

Reply:
Changed documentation.

Reply:
3rd time's the charm. Changed